draft-scenarios/clash/dragon_valley.tex

```diff
@@ -0,0 +1,153 @@
+\textbf{Starting Resources:} 14 \svg{gold}, 4 \svg{building_materials}, 1 \svg{valuables}
+
+\textbf{Starting Income:} 10 \svg{gold}, 0 \svg{building_materials}, 0 \svg{valuables}
+
+\textbf{Starting Units:}
+\begin{itemize}
+  \item A Few of \svgunit{bronze} Units of the player's choice
+  \item A Pack of \svgunit{bronze} Units of the player's choice
+\end{itemize}
+
+\textbf{Town Buildings:}
+\begin{itemize}
+  \item \svgunit{bronze} Dwelling
+  \item Each player may build the the \svg{building_special_tent} building with a discount of 4 \svg{gold} and 2 \svg{building_materials}.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Map Setup}}
+Take the following Map Tiles and arrange them as shown in the Scenario map layout ($P$ stands for the number of players):
+
+\begin{itemize}
+  \item P × Starting (I) Map Tile
+  \item 3P × Far (II-III) Map Tiles - one per player must include a Settlement, and one per player must include a Trading Post
+  \item 2P × Near (IV-V) Map Tile
+  \item P × Center (VI-VII) Map Tiles (C5 is treated as a Settlement)
+\end{itemize}
+
+\textbf{Additionally, for 2--3 players:}
+\begin{itemize}
+  \item 1 × Center (VI-VII) Map Tile with a Dragon Utopia
+\end{itemize}
+
+\textbf{For 4 players:}
+\begin{itemize}
+  \item 2 × Center (VI-VII) Map Tiles with a Dragon Utopia
+\end{itemize}
+
+\subsection*{\MakeUppercase{Victory Conditions}}
+The game ends at the end of the Round when any of these conditions are met:
+
+\begin{itemize}
+ \item One player has defeated all other players' Main Heroes once - \textit{That player wins the game immediately.}
+ \item One player conquers both Dragon Utopias %- \textit{That player wins the game immediately.}
+ \item At the end of Round 13 (or Round 9 for the shorter game).
+\end{itemize}
+
+\subsection*{\MakeUppercase{Victory Points}}
+If no player has achieved an immediate victory, the player with the most Victory Points (VP) wins:
+
+\begin{itemize}
+ \item 5 VP for conquering a Dragon Utopia
+ \item 4 VP for controlling 5 Mines or Settlements on Near and Center Tiles - \textit{once per game}
+ \item 4 VP for defeating an enemy's Main Hero - \textit{once per defeated Faction}
+ \item 2 VP for defeating an enemy's Secondary Hero - \textit{once per defeated Faction}
+ \item 2 VP for each enemy Main Town captured - \textit{once per captured Faction}
+ \item 1 VP for every controlled Mine or Settlement
+% \item 1 VP for every Level of Experience of your Main Hero
+\end{itemize}
+
+\subsection*{\MakeUppercase{Timed Events}}
+
+\begin{itemize}
+  \item Start of \textbf{\nth{1} Round:} All Heroes gain +1 \svgeven{movement}.
+  \item Start of \textbf{\nth{4} Round:} Remove black Cubes from all locations that give resources, morale or Cards (not from Grail).
+  \item Start of \textbf{\nth{8} Round:} Repeat event of Round 4.
+  \item Start of \textbf{\nth{9} Round:} Repeat event of Round 1.
+  \item End of \textbf{\nth{10} Round:} Player(s) with lowest XP roll 2 \svg{resource} dice.
+\end{itemize}
```

Comment:
```suggestion
\begin{itemize}
  \item[\textbf{\nth{1}}] \textbf{Round:} All Heroes gain +1 \svgeven{movement}.
  \item[\textbf{\nth{4}}] \textbf{Round:} Remove Black Cubes from all locations that give Resources, Morale or Cards (but not from the Grail).
  \item[\textbf{\nth{8}}] \textbf{Round:} Repeat event of Round 4.
  \item[\textbf{\nth{9}}] \textbf{Round:} Repeat event of Round 1.
  \item[\textbf{End}] \textbf{of \nth{10} Round:} Player(s) with lowest XP roll 2 \svg{resource} dice.
\end{itemize}
```
Since you've made it so compact, let's go with it. There's no need for "start of", as this is the default.